torch/_refs/__init__.py

```diff
@@ -2188,13 +2199,20 @@ def _make_copy_from_view(fn):
     Given a view function (e.g. torch.diagonal) generates its copy variant (e.g. torch.diagonal_copy)
     """
     name = fn.__name__
-    fn = out_wrapper()(fn)
+    op = getattr(aten, name)
+    if "out" in op.overloads():
```

Comment:
Also, this shouldn't look into the `op` overloads, but into the `op_copy` overloads.